pay-api/src/pay_api/resources/v1/eft_short_names.py

```diff
@@ -171,3 +171,25 @@ def post_eft_shortname_link(short_name_id: int):
 
     current_app.logger.debug('>post_eft_shortname_link')
     return jsonify(response), status
+
+
+@bp.route('/<int:short_name_id>/links/<int:short_name_link_id>', methods=['DELETE', 'OPTIONS'])
+@cross_origin(origins='*', methods=['DELETE'])
+@_jwt.has_one_of_roles([Role.SYSTEM.value, Role.MANAGE_EFT.value])
+def delete_eft_shortname_link(short_name_id: int, short_name_link_id: int):
```

Comment:
Basic UNLINK route currently used for just removing a link that was created in error before the 6pm processing that is still in PENDING status.

This will be expanded upon for refund flow.